lib/mock.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Janus.Mock do
```

Comment:
I am not the fan of this name, but I couldn't figure out better at this time. Open to discussion on this topic.

Reply:
Seems fine, especially as it has `moduledoc false`